pkg/net/net_linux.go

```diff
@@ -163,3 +166,56 @@ func netParseEthtoolFeature(line string) *NICCapability {
 		CanEnable: !fixed,
 	}
 }
+
+func netDeviceHWAddress(netDevDir, netDevName string) NICAddress {
```

Comment:
and I think it would be best to just name this function netDevicePCIAddress() since the function only looks up the PCI address (and have it return a `*string`)

Reply:
sure, on it.